skimage/_shared/kahan.pyx

```diff
@@ -0,0 +1,86 @@
+# -*- python -*-
+#cython: cdivision=True
+#cython: boundscheck=False
+#cython: nonecheck=False
+#cython: wraparound=False
+
+import numpy as np
+import cython
+
+
+def fsum_vector(cython.floating[:] v, int calc_cumsum=0):
```

Comment:
call this function ``_fsum_vector`` if it's not meant to be called outside of submodule?